README.md

```diff
@@ -0,0 +1 @@
+# chkconfig | [![Build Status](https://github.com/fedora-sysv/chkconfig/workflows/Integration%20tests/badge.svg)]
```

Comment:
wanna throw in the Actions build badge too?

Reply:
Yes, I would like to have build status easily visible. Is it correct syntax?

Reply:
huh for some reason I misread the link there and thought it was going to travis, I'm blind.
So the syntax is correct, but I"d modify it so people can click on it and go to the build status page:
```
[![Build Status](https://github.com/fedora-sysv/chkconfig/workflows/Integration%20tests/badge.svg)](https://github.com/fedora-sysv/chkconfig/actions?query=workflow%3AIntegration+tests)
```
ala:

[![Build Status](https://github.com/fedora-sysv/chkconfig/workflows/Integration%20tests/badge.svg)](https://github.com/fedora-sysv/chkconfig/actions?query=workflow%3AIntegration+tests)

(won't be a useful link until after this is merged though)